src/rules/lowercase-name.ts

```diff
@@ -48,7 +34,9 @@ const hasStringAsFirstArgument = (
 const isJestFunctionWithLiteralArg = (
   node: TSESTree.CallExpression,
 ): node is CallExpressionWithCorrectCalleeAndArguments =>
-  isItTestOrDescribeFunction(node) && hasStringAsFirstArgument(node);
+  (isTestCase(node) || isDescribe(node)) &&
+  ['it', 'test', 'describe'].includes(node.callee.name) &&
```

Comment:
doesn't `(isTestCase(node) || isDescribe(node))` cover this line?

Reply:
I was going to ask you about that :)

Previously the check was just for these three functions, and the description & tests for the rule only focus on those three:

> Enforce `it`, `test` and `describe` to have descriptions that begin with a lowercase letter. This provides more readable test failures.

I think it should be fine to remove that line, but wanted to implement the original behaviour + ask first.

Reply:
ah right, since we'd now get `xtest`, `ftest` etc? I think it's fine to detect all - they're just modifiers on the same functions

Reply:
We can do that in a follow-up, though